HOW_WE_USE_GITHUB.md

```diff
@@ -91,83 +95,110 @@ At the most basic "bird's eye view" level, sorted issues will fall into the cate
 
+### Who does the sorting?
 
-#### How do items show up for sorting?
+Sorting engineers are a conda governance [sub-team][sub-team]; they are a group of Anaconda, Inc. and community members who are responsible for making decisions regarding closing issues and setting feature work priorities, among other sorting-related tasks.
 
-New issues that are opened in any of the repositories in the [`conda` GitHub project][conda-org] will show up in the `Sorting` view of the [Planning project][project-planning]. This process is executed via [GitHub Actions][docs-actions]. The two main GitHub Actions workflows utilized for this purpose are [Issues][workflow-issues] and [Project][workflow-project].
+### How do items show up for sorting?
 
-The GitHub Actions in the `conda/infra` repository are viewed as canonical; the [Sync workflow][workflow-sync] sends out any modifications to other `conda` repositories from there.
+New issues that are opened in any of the repositories in the [conda GitHub project][conda-org] will show up in the "Sorting" tab of the [Planning project][project-planning]. There are two [GitHub Actions][docs-actions] workflows utilized for this purpose; [`.github/workflows/issues.yml`][workflow-issues] and [`.github/workflows/projet.yml`][workflow-project].
 
+The GitHub Actions in the [`conda/infrastructure`][infrastructure] repository are viewed as canonical; the [`.github/workflows/sync.yml` workflow][workflow-sync] sends out any modifications to other `conda` repositories from there.
 
-#### What is done about the issues in "sorting" mode?
+### What is done about the issues in the "Sorting" tab?
 
-Issues in the ["Sorting" tab of the project board][project-sorting] have been reviewed by a sorting engineer and are considered ready for the following procedures:
+Issues in the ["Sorting" tab of the project board][project-sorting] are considered ready for the following procedures:
 
 - Mitigation via short-term workarounds and fixes
 - Redirection to the correct project
 - Determining if support can be provided for errors and questions
 - Closing out of any duplicate/off-topic issues
 
-The sorting engineers on rotation are not seeking to _resolve_ issues that arise. Instead, the goal is to understand the ticket and to determine whether it is an issue in the first place, and then to collect as much relevant information as possible so that the maintainers of `conda` can make an informed decision about the appropriate resolution schedule.
+The sorting engineers on rotation are not seeking to _resolve_ issues that arise. Instead, the goal is to understand the ticket and to determine whether it is an issue in the first place, and then to collect as much relevant information as possible so that the maintainers can make an informed decision about the appropriate resolution schedule.
```

Comment:
```suggestion
The sorting engineers on rotation are not seeking to _resolve_ issues that arise. Instead, the goal is to understand the issue and to determine whether it is legitimate, and then to collect as much relevant information as possible so that the maintainers can make an informed decision about the appropriate resolution schedule.
```